### DISCUSSION TITLE
Prompt textbox is getting corrupted in an odd and inconsistent way (never seen before `v0.3.35`)

### DISCUSSION THREAD

**Discussion by rotemdan** [open-webui/open-webui#6477] [Issues]
![Animation](https://github.com/user-attachments/assets/5d13b6ea-9174-4dd5-82d4-99be37224384)

Seems like something about the asterisk characters is causing it to go crazy? Is there some sort WYSIWYG functionality newly added that may be causing this? I've used `open-webui` a lot, and I've never seen this happening until the very recent updates.

In other cases I've recently seen, multiple times that in a particular state of the editor, pasting code would paste it without leading whitespace (spaces and tabs). I'll send a GIF for it as well once I encounter it again.

**Comment by silentoplayz** [comment]
If you're using [v0.3.34](https://github.com/open-webui/open-webui/releases/tag/v0.3.34) or the latest version of Open WebUI, you should be able to toggle off `Rich Text Input for Chat` from the `Interface` Settings menu.
> 🎨 Rich Text Input Toggle: Users can now switch back to legacy textarea input for chat if they prefer simpler text input, though rich text is still the default until deprecation.
![image](https://github.com/user-attachments/assets/8ae66c2a-70b7-48e2-bc92-8f2d6c5c2e0b)

**Comment by rotemdan** [comment]
Thanks. This can be useful to anyone else encountering it. I didn't know it was a beta feature, it just became enabled by itself at some point.

I hope turning the feature off also solves the issue with the missing whitespace when pasting code.

I personally prefer simple raw plain text as input, since rich-text editing, is usually very hard to get right (to a usable, stable experience), especially when source code and math are involved.
